charts/hawkbit/templates/servicesimpleui.yaml

```
@@ -12,10 +12,10 @@ metadata:
 spec:
   type: {{ .Values.serviceSimpleUI.type }}
   ports:
-    - port: {{ .Values.ingress.hosts_hawkbit_simple_ui.port }} #port expose pour ingress
-      targetPort: {{ .Values.serviceSimpleUI.port }} ## port of the pod/deployement
+    - port: {{ .Values.serviceSimpleUI.port }}
+      targetPort: http
```

```suggestion
      targetPort: 8088
```
vue que c'est le port du docker de l'ui

---

pas besoin